src/iosMain/kotlin/qrcode/render/QRCodeGraphics.ios.kt

```diff
@@ -69,7 +69,14 @@ actual open class QRCodeGraphics actual constructor(
      * @see availableFormats
      */
     actual open fun getBytes(format: String): ByteArray =
-        ByteArray(0)
+        (nativeImage() as? UIImage)?.let { image ->
+            when (format) {
+                "HEIC" -> UIImageHEICRepresentation(image)
+                "JPEG" -> UIImageJPEGRepresentation(image, 1.0)
+                "PNG" -> UIImagePNGRepresentation(image)
+                else -> null
+            }?.toByteArray() ?: ByteArray(0)
+        } ?: ByteArray(0)
```

Comment:
🤦 

I'll add a triple-check of this sort of stuff for the next release, sorry x_x